Sources/OceanComponents/ComponentsSwiftUI/Alert/OceanSwiftUI+Alert.swift

```diff
@@ -23,7 +23,7 @@ extension OceanSwiftUI {
         @Published public var actionType: ActionType
         @Published public var tooltipText: String
         @Published public var hasCornerRadius: Bool
-        @Published public var brands: Brands?
+        @Published public var brands: OceanSwiftUI.BrandsParameters?
```

Comment:
```suggestion
        @Published public var brands: BrandsParameters?
```

precisa do OceanSwiftUI?